ksqldb-tools/src/main/java/io/confluent/ksql/tools/migrations/commands/ApplyMigrationCommand.java

```diff
@@ -313,26 +313,74 @@ private void executeCommands(
       final Clock clock,
       final String previous
   ) {
-    final Map<String, Object> properties = new HashMap<>();
     final List<String> commands = CommandParser.splitSql(migrationFileContent);
 
+    executeCommands(
+        commands, ksqlClient, config, executionStart, migration, clock, previous, true);
+    executeCommands(
+        commands, ksqlClient, config, executionStart, migration, clock, previous, false);
+  }
+
+  private void executeCommands(
+      final List<String> commands,
+      final Client ksqlClient,
+      final MigrationConfig config,
+      final String executionStart,
+      final MigrationFile migration,
+      final Clock clock,
+      final String previous,
+      final boolean verify
```

Comment:
```suggestion
      final boolean validateOnly
```
to clarify that this boolean param being set to true performs less than when it is set to false, not to the other way around (as the name `verify` currently suggests).